Docs/userguide/quantization_workflow.rst

```diff
@@ -89,70 +39,68 @@ Supported precisions to run inference on target runtimes like |qnn|_ are:
 
-Set the bit-width to 16 bits for both weights and activations when creating the QuantSim.
-Then, obtain the off-target quantized accuracy metric for the quantized model and verify if
-it aligns with the FP32 model. If it doesn't, please report an issue to |aimet|_.
+Then, obtain the off-target quantized accuracy metric for the quantized model and verify that it agrees with the FP32 model. If it does not, you can help improve AIMET by reporting an issue to |aimet|_.
 
-Apply PTQ or QAT at specified precision
----------------------------------------
+5. Applying PTQ or QAT
+----------------------
 
-If any of the metrics are not acceptable with higher precision, begin with weights at
-INT8 precision and activations at INT16 precision. In this step, before creating the QuantSim,
-ensure that the FP32 model adheres to model specific guidelines. For instance, in PyTorch,
-QuantSim can only quantize math operations performed by :class:`torch.nn.Module` objects, while
-:class:`torch.nn.functional` calls will be incorrectly ignored. Please refer to framework specific
-pages to know more about such model guidelines.
+If any of the metrics are not acceptable with higher precision, begin with weights at INT8 precision and activations at INT16 precision. 
 
-If the off-target quantized accuracy metric is not meeting expectations, you can use PTQ or QAT
-techniques to improve the quantized accuracy for the desired precision. The decision between
-PTQ and QAT should be based on the quantized accuracy and runtime needs.
+If the off-target quantized accuracy metric does not meet expectations, use PTQ or QAT techniques to improve the quantized accuracy for the implemented precision. The decision to use PTQ or QAT should be based on your quantized accuracy and runtime needs.
 
 .. image:: ../images/quantization_workflow_5.png
 
-Once the off-target quantized accuracy metric is satisfactory, proceed to :ref:`evaluate the
-on-target metrics<opt-guide-on-target-inference>` at this precision. If the on-target metrics
-still do not meet the your requirements, consider further reducing the precision
-(for example W8A8, W4A8) and repeat the current step.
+Next: deploying the model
+-------------------------
 
-Deploy
-------
+Once the off-target quantized accuracy is satisfactory, proceed to :ref:`evaluate the
+on-target metrics<opt-guide-on-target-inference>` at this precision. If the on-target metrics still do not meet your requirements, consider further reducing the precision (for example to W8A8 or W4A8) and repeat the application of PTQ or QAT to optimize the model.
 
-Once the quantized accuracy and runtime requirements are achieved at the desired precision,
-the optimized model is ready for deployment on the target runtimes.
+Once the quantized accuracy and runtime requirements are achieved at the desired precision, deploy the optimized model on the target runtimes.
```

Comment:
Can we provide explicit instructions for how to deploy the model, or a link to those instructions?